mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -67,9 +138,52 @@ AUI.add(
 								}
 							);
 
+							var taskHist = A.Lang.sub(
+								TPL_HIST,
+								{
+									task: taskInput.val()
+								}
+							);
+
 							todoList.append(taskHtml);
 
 							taskInput.val('');
+
+							var taskList = instance.one('.task-list');
```

Comment:
There is a lot of code duplication with this block and line 77. Reusable code should be extracted to a function, abstracted in functionality.